### PR TITLE
Epistemics Job Localizations

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

Part of our Maintainer meeting where we decided we wished to go "Completely all-out" with the "Science Cult" side of Epistemics. Next up, extremely fancy white robes. This PR localizes the various jobs in Epistemics to follow the "Science Cult" theme. Research Assistant became Novice, Scientist became Acolyte, Senior Researcher became Mystic, and Roboticist became Golemancer. 

So now the Epistemics roles are:
Novice, Acolyte, Mystic, Golemancer, Chaplain, Psionic Mantis, Cataloguer, and Mystagogue.

image

# Changelog

:cl:
- add: Added localizations for all Epistemics jobs.

### CODE REVIEW

**Comment by FoxxoTrystan** [comment]
I am compelatly not a fan of this.

File: Resources/Locale/en-US/job/job-names.ftl
Comment (gluesniffler) [inline Resources/Locale/en-US/job/job-names.ftl:1]: Novice is too vague, should be Novice Acolyte, Acolyte Initiate or something like that.

Comment (VMSolidus) [reply Resources/Locale/en-US/job/job-names.ftl:1]: I'd prefer one-word, so how about we do Noviciate?

Comment (gluesniffler) [reply Resources/Locale/en-US/job/job-names.ftl:1]: works ig

**Comment by VMSolidus** [comment]
> I am compelatly not a fan of this.

Well, me and Death aren't going to tolerate having a "Half-ass it both ways" Epistemics. Science is blatantly unfinished, Epistemics is blatantly unfinished. One of these things is waaaay cooler than the other.

**Comment by stellar-novas** [comment]
Side note, cataloguer needs a new job icon. They're more solidly an Epistemics role and they look weird being grey.

**Comment by VMSolidus** [comment]
> Side note, cataloguer needs a new job icon. They're more solidly an Epistemics role and they look weird being grey.

image

I actually just fixed it too.

File: Resources/Locale/en-US/job/job-names.ftl [hunk 1]
```diff
@@ -86,17 +87,17 @@ JobPassenger = Passenger
 JobPsychologist = Psychologist
 JobQuartermaster = Logistics Officer
 JobReporter = Reporter
-JobResearchAssistant = Research Assistant
+JobResearchAssistant = Novice
```
Comment (stellar-novas) [inline Resources/Locale/en-US/job/job-names.ftl:90]:
```suggestion
JobResearchAssistant = Noviciate
```

**Review by stellar-novas — APPROVED** [review]
It would be nice if we could do time gated job titles through loadouts. That way everyone could be a mystic/acolyte if everyone qualifies.

**Comment by Mnemotechnician** [comment]
In my opinion, this is a bad idea. From my own experience (over 400 hours as a scientist/mysta across two EE forks), epistemics are 95% science and 5% religion/psionics. In particular, most scientists I've seen on floof would never ever interact with psionics, besides just occasionally rolling a power and going "oh cool, I have that now".

**Comment by VMSolidus** [comment]
Let me cook goddamnit.

**Comment by Remuchi** [comment]
It is a great idea in the long-run. But at the moment psionics simple doesn't have enough content to run the whole department around it. The only person somehow interacting with psionics part of the department is Mantis. Yeah the chaplain sometimes works with Oracle but it's not really important to the department as whole?

**Review by Peptide90 — APPROVED** [review]

**Review by FoxxoTrystan — APPROVED** [review]
Il approve code wise...

**Pull request merged**